src/tscpp/util/TextView.cc

```diff
@@ -64,31 +64,31 @@ strcasecmp(const std::string_view &lhs, const std::string_view &rhs)
   return zret;
 }
 
+const int8_t ts::svtoi_convert[256] = {
+  /* [can't do this nicely because clang format won't allow exdented comments]
```

Comment:
spelling of extended

Reply:
No, it's really "exdented", the opposite of "indented".